cinema/views.py

```diff
@@ -127,6 +132,38 @@ def upload_image(self, request, pk=None):
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                name="title",
+                type={"type": "array", "items": {"type": "number"}},
```

Comment:
The type for the 'title' parameter should be 'string' instead of 'array' since it is used for filtering by title, which is a text field.